c2corg_ui/views/document.py

```diff
@@ -221,3 +222,13 @@ def _diff(self):
             return self.template_input
 
         raise HTTPNotFound()
+
+    notFound = 'c2corg_ui:templates/404.html'
+
+    @view_config(context=HTTPInternalServerError, renderer=notFound)
+    @view_config(context=HTTPBadRequest, renderer=notFound)
+    @notfound_view_config(renderer=notFound)
+    def notfound(self):
```

Comment:
This view is not only used document pages, right? So maybe it would make sense to move this declaration from `views/document.py` to `views/__init__.py`?